www/js/services/incidents_services.js.BASE.5920.js

```diff
@@ -0,0 +1,243 @@
+angular.module('mobility')
```

Comment:
I'm a little confused on the naming scheme. What is the significance of "Base.5920.js"?